acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-lines-with-vat-adjustments.feature

```diff
@@ -0,0 +1,136 @@
+# For MODINVOICE-554
+Feature: Check invoice lines with VAT adjustments
+
+  Background:
+    * url baseUrl
+
+    * call login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+  @Positive
+  Scenario: Check invoice lines with VAT adjustments
+    # Check that each invoice line VAT calculation is unaffected by other invoice lines
+    # as other forms of adjustments typically involve every invoice line in its adjustment calculation
+    * def fundId = call uuid
+    * def budgetId = call uuid
+
+    * def adjustmentId = call uuid
+    * def invoiceId1 = call uuid
+    * def invoiceId2 = call uuid
+    * def invoiceLineId1 = call uuid
+    * def invoiceLineId2 = call uuid
+    * def invoiceLineId3 = call uuid
+    * def invoiceLineId4 = call uuid
+
+    ### 1. Create fund and budgets
+    * configure headers = headersAdmin
+    * def v = call createFund { id: '#(fundId)', code: '#(fundId)' }
+    * def v = call createBudget { id: '#(budgetId)', fundId: '#(fundId)', allocated: 1000 }
+    * configure headers = headersUser
+
+    ### 4. Create invoices
+    * def invoiceAdjustments =
+      """
+      [{
+        "id": "#(adjustmentId)",
+        "value": 7.0,
+        "type": "Percentage",
+        "prorate": "By amount",
+        "description": "VAT",
+        "relationToTotal": "Included in",
+        "fundDistributions": [],
+        "exportToAccounting": false
+      }]
+      """
+    * table invoices
+      | id         | invoiceId  | currency | adjustments        |
+      | invoiceId1 | invoiceId1 | 'EUR'    | invoiceAdjustments |
```

Comment:
Two invoices used, first invoice is used to verify the logic for a simple single-line case, while the second invoice for cases with multiple invoice lines being added and/or removed.  Both cases should not affect other invoice line adjustments.